### PR TITLE
Remove rawgit

### PR DESCRIPTION
**Pull request by ycaokris**
- [x] Remove rawgit and add assets folder
- [x] gunicorn run local successfully

### CODE REVIEW

File: assets/ga.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```
Comment (shammamah) [inline assets/ga.js]: Not sure if we still want this in the app. Leave it in for now, but open an issue in the repo!

Comment (ycaokris) [reply assets/ga.js]: Discussed with @xhlulu This app doesn't need GA javascript, it runs unaffected with removing it.

**Comment by nicolaskruchten** [comment]
Wait why are we removing Google Analytics tags? They’re there for tracking traffic!

**Comment by shammamah** [comment]
@nicolaskruchten None of the other apps have it, so I assumed it was left there by accident. We can add it back in if that isn't the case!

**Comment by nicolaskruchten** [comment]
I would imagine actually that we would want all apps to have these. @jemprice please confirm: should dash demo apps have GA tags?

**Comment by jemprice** [comment]
Yes, Please!

**Pull request merged**